src/test/hello.test.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { expect } from "chai";
+
+describe("Query hello", () => {
+  const query = `
+        query Hello {
+          hello
+        }
+      `;
+  it("should return hello word", async () => {
+    const urlDB = "http://localhost:4000//graphql";
```

Comment:
a url fica junto do describe, afinal em todo o teste vc vai usar o mesmo post. O que fica dentro do it é coisa que é apenas dele.